### PR TITLE
news-server-5

### PR DESCRIPTION
**Pull request by TomMirfin**
News server 5 first commit

### CODE REVIEW

File: __tests__/endpoint.test.js [hunk 1]
```diff
@@ -82,3 +82,40 @@ describe("/api/articles/", () => {
...
+        expect(body).toBeSortedBy(body.created_at, { descending: true });
+      });
+  });
+  test(`Each reply must respond with an object which includes author, title, article_id, topic, created_at, votes, article_img_url and comment count `, () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          console.log(body);
```
Comment (Hard-Luck) [inline __tests__/endpoint.test.js:101]: remove this

File: __tests__/endpoint.test.js [hunk 2]
```diff
@@ -82,3 +82,40 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test(`when all articles is requested, reponds with all articles in ascending order by creating date and no body`, () => {
+    return request(app)
+      .get("/api/articles")
```
Comment (Hard-Luck) [inline __tests__/endpoint.test.js:88]: we would also like a length check to make sure our toBeSortedBy is checking a populated array as an empty array or an array with 1 thing in would always be sorted by default

File: __tests__/endpoint.test.js [hunk 3]
```diff
@@ -82,3 +82,40 @@ describe("/api/articles/", () => {
...
+      });
+  });
+  test(`Each reply must respond with an object which includes author, title, article_id, topic, created_at, votes, article_img_url and comment count `, () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          console.log(body);
+          expect(typeof article.author).toBe("string");
```
Comment (Hard-Luck) [inline __tests__/endpoint.test.js:102]: same here the length check makes sure that your expects in the forEach are being triggered

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
Couple of changes to be made, see my comments

**Review by mezzDavies — APPROVED** [review]
Approved - merge away!

**Pull request merged**